Index: build/resources/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Database connection settings\nspring.datasource.url=jdbc:mysql://localhost:3306/beatbuzz\nspring.datasource.username=beatbuzz\nspring.datasource.password=Music4Ever!\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\n\n\n# Specify the DBMS\nspring.jpa.database = MYSQL\n\n# Show or not log for each sql query\nspring.jpa.show-sql = false\n\n# Hibernate ddl auto (create, create-drop, update)\nspring.jpa.hibernate.ddl-auto = update\n\n# Use spring.jpa.properties.* for Hibernate native properties (the prefix is\n# stripped before adding them to the entity manager)\nspring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect\n\n# App Properties\nbeatbuzz.app.jwtSecret= ======================beatbuzz=Spring===========================\nbeatbuzz.app.jwtExpirationMs=86400000
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/build/resources/main/application.properties b/build/resources/main/application.properties
--- a/build/resources/main/application.properties	
+++ b/build/resources/main/application.properties	
@@ -18,6 +18,6 @@
 # stripped before adding them to the entity manager)
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
 
-# App Properties
-beatbuzz.app.jwtSecret= ======================beatbuzz=Spring===========================
-beatbuzz.app.jwtExpirationMs=86400000
\ No newline at end of file
+## App Properties
+#beatbuzz.app.jwtSecret= ======================beatbuzz=Spring===========================
+#beatbuzz.app.jwtExpirationMs=86400000
\ No newline at end of file
