Index: src/main/java/org/launchcode/config/CorsConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.launchcode.config;\n\nimport org.springframework.context.annotation.Configuration;\nimport org.springframework.web.servlet.config.annotation.CorsRegistry;\nimport org.springframework.web.servlet.config.annotation.WebMvcConfigurer;\n\n@Configuration\npublic class CorsConfig implements WebMvcConfigurer {\n\n    @Override\n    public void addCorsMappings(CorsRegistry registry) {\n        registry.addMapping(\"/**\")\n                .allowedOrigins(\"http://localhost:3000\") // Update with your frontend URL\n\n                .allowedMethods(\"GET\", \"POST\", \"PUT\", \"DELETE\")\n                .allowedHeaders(\"*\");\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/launchcode/config/CorsConfig.java b/src/main/java/org/launchcode/config/CorsConfig.java
--- a/src/main/java/org/launchcode/config/CorsConfig.java	(revision 6157803ad1f55e211e80b1a1fbf839c62292d66a)
+++ b/src/main/java/org/launchcode/config/CorsConfig.java	(date 1713653899057)
@@ -10,9 +10,9 @@
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/**")
-                .allowedOrigins("http://localhost:3000") // Update with your frontend URL
+                .allowedOrigins("http://localhost:3001") // Update with your frontend URL
 
                 .allowedMethods("GET", "POST", "PUT", "DELETE")
                 .allowedHeaders("*");
     }
-}
+};
Index: build/resources/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Database connection settings\nspring.datasource.url=jdbc:mysql://localhost:3306/beatbuzz\nspring.datasource.username=beatbuzz\nspring.datasource.password=Music4Ever!\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\n\n\n# Specify the DBMS\nspring.jpa.database = MYSQL\n\n# Show or not log for each sql query\nspring.jpa.show-sql = false\n\n# Hibernate ddl auto (create, create-drop, update)\nspring.jpa.hibernate.ddl-auto = update\n\n# Use spring.jpa.properties.* for Hibernate native properties (the prefix is\n# stripped before adding them to the entity manager)\nspring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/build/resources/main/application.properties b/build/resources/main/application.properties
--- a/build/resources/main/application.properties	(revision 6157803ad1f55e211e80b1a1fbf839c62292d66a)
+++ b/build/resources/main/application.properties	(date 1713653899056)
@@ -9,7 +9,7 @@
 spring.jpa.database = MYSQL
 
 # Show or not log for each sql query
-spring.jpa.show-sql = false
+spring.jpa.show-sql = true
 
 # Hibernate ddl auto (create, create-drop, update)
 spring.jpa.hibernate.ddl-auto = update
