Index: build/resources/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Database connection settings\nspring.datasource.url=jdbc:mysql://localhost:3306/beatbuzz\nspring.datasource.username=beatbuzz\nspring.datasource.password=Music4Ever!\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\n\n\n# Specify the DBMS\nspring.jpa.database = MYSQL\n\n# Show or not log for each sql query\nspring.jpa.show-sql = false\n\n# Hibernate ddl auto (create, create-drop, update)\nspring.jpa.hibernate.ddl-auto = update\n\n# Use spring.jpa.properties.* for Hibernate native properties (the prefix is\n# stripped before adding them to the entity manager)\nspring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/build/resources/main/application.properties b/build/resources/main/application.properties
--- a/build/resources/main/application.properties	(revision f738df0507cf9cfb0f353c7582b5076f4f1a1ec7)
+++ b/build/resources/main/application.properties	(date 1713226833662)
@@ -9,7 +9,7 @@
 spring.jpa.database = MYSQL
 
 # Show or not log for each sql query
-spring.jpa.show-sql = false
+spring.jpa.show-sql = true
 
 # Hibernate ddl auto (create, create-drop, update)
 spring.jpa.hibernate.ddl-auto = update
